src/Export/Manager.php

```diff
@@ -129,6 +129,19 @@ public function exportLocalEntity(
         )
       );
     }
+
```

Comment:
We need to call the field mapping `fieldMapping ()` function here just like we're doing with the entity mapping.